common/lib/xmodule/xmodule/word_cloud_module.py

```diff
@@ -37,8 +45,12 @@ def pretty_bool(value):
     return value in bool_dict
 
 
-class WordCloudFields(object):
-    """XFields for word cloud."""
+class WordCloudBlock(  # pylint: disable=abstract-method
+    EmptyDataRawMixin, XmlMixin, EditingMixin,
+    XModuleDescriptorToXBlockMixin, XModuleToXBlockMixin, HTMLSnippet, ResourceTemplates, XModuleMixin,
```

Comment:
Nit: If we put these on their own lines, like the `import`s above, it reduces the churn and subsequent chance of conflicts if/when we modify this `class`.